src/api/app/models/bs_request.rb

```diff
@@ -272,6 +272,16 @@ def history_elements
     HistoryElement::Base.where(id: request_history_elements.pluck(:id) + review_history_elements.pluck(:id)).order(:created_at)
   end
 
+  def history_elements_accepted_or_declined
```

Comment:
This shouldn't be in the `BsRequest` model. We should instead use a query object. I have created a [card](https://trello.com/c/rbez8hTw/2658-use-query-objects-for-complex-database-queries) in our backlog to follow up on this.

Reply:
I've removed it. It's not used anymore